tests/utils/common.go

```diff
@@ -75,6 +96,9 @@ func IsHostpathProvisioner() bool {
 
 // AddProvisionOnNodeToAnn adds 'kubevirt.io/provisionOnNode' annotaion
 func AddProvisionOnNodeToAnn(pvcAnn map[string]string) {
+	if DefaultNodeName == "" {
+		ginkgo.Skip("No scheduleable nodes available")
```

Comment:
Do you think it is cleaner to add the skip call to the test function itself rather than a helper?  Is that too cumbersome?

Reply:
So this is called by 40 or so tests I believe, so doing it individually on each is going to be cumbersome and somewhat error prone.